undertow/EAP7-858_Session_Manager_Operations.adoc

```diff
@@ -0,0 +1,95 @@
+= EAP7-858 Session Manager Operations
+:author:            Stuart Douglas
+:email:             sdouglas@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+This feature is to add additional features to the management model to allow additional operations to be
+performed on the web session manager.
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-858[EAP7-858]
+
+=== Related Issues:
+
+* {issue-base-url}/WFLY-9540[WFLY-9540]
+* {issue-base-url}/PRODMGT-1799[PRODMGT-1799]
+
+=== Dev Contacts:
+
+* mailto:{email}[{author}]
+
+=== QE Contacts:
+
+=== Affected Projects or Components:
+
+* Wildfly
+
+== Requirements
+
+This will be implemented as additional operations under the `deployment=*/subsystem=undertow` management model address.
+
+The extra operations implemented will be:
```

Comment:
I can see no equivalent function to `createEmptySession`. Is this intentional?

Reply:
Yes, I don't really see any use for for createEmptySession()